test/worker/xml/test_validate_xml.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+from lxml import etree
+
+from workers.default.xml.xml_validator import validate_xml
+
+
+class ValidateXMLTest(unittest.TestCase):
+
+    ojs_xml_file = 'test/resources/objects/xml/ojs_import.xml'
```

Comment:
eventuell nutzt du hier unsere env variable?
```
resource_dir = os.environ['RESOURCE_DIR']
```
Gibt ja einen Grund, dass es die gibt..

Reply:
Ja klingt gut. Aber wo wird RESOURCE_DIR eigentlich gesetzt? Finde es in der .env nicht.